arbitrary-file-write/file-overwrite/README.md

```diff
@@ -3,4 +3,86 @@
 * as root or user you can overwrite a file and ...
 	* fully control content
 	* partially control content (eg. parts of a logfile)
-	* none (empty file)
\ No newline at end of file
+	* none (empty file)
+
+### pam
+If we know enough about the target os we can overwrite their user authentication system logic with a backdoor. Here's an [example](https://infosecwriteups.com/creating-a-backdoor-in-pam-in-5-line-of-code-e23e99579cd9) that returns a successful login for a specific hardcoded password. 
```

Comment:
I think it would be useful to have the "stackoverflow" policy, where when you reference a blog, make sure to quote all the relevant pieces here as well, in case the link goes dead.